### PR TITLE
Correct callback signature for express-handlebars renderView

### PR DESCRIPTION
**Pull request by chrisgedrim**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [renderView](https://github.com/ericf/express-handlebars/blob/master/lib/express-handlebars.js#L181) [calls](https://github.com/ericf/express-handlebars/blob/master/lib/express-handlebars.js#L235-L236) one of two [util methods](https://github.com/ericf/express-handlebars/blob/master/lib/utils.js#L15-L29)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by stpettersens** [comment]
Approve

On 6 April 2019, at 23:22, TypeScript Bot <notifications@github.com> wrote:

@chrisgedrim Thank you for submitting this PR!

🔔 @stpettersens @yhaskell - please review this PR in the next few days. Be sure to explicitly select Approve or Request Changes in the GitHub UI so I know what's going on.

If no reviewer appears after a week, a DefinitelyTyped maintainer will review the PR instead.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.￼

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/DefinitelyTyped/DefinitelyTyped","title":"DefinitelyTyped/DefinitelyTyped","subtitle":"GitHub repository","main_image_url":"<a href="https://github.githubassets.com/images/email/message_cards/header.png">https://github.githubassets.com/images/email/message_cards/header.png</a>","avatar_image_url":"<a href="https://github.githubassets.com/images/email/message_cards/avatar.png">https://github.githubassets.com/images/email/message_cards/avatar.png</a>","action":{"name":"Open in GitHub","url":"<a href="https://github.com/DefinitelyTyped/DefinitelyTyped">https://github.com/DefinitelyTyped/DefinitelyTyped</a>"}},"updates":{"snippets":[{"icon":"PERSON","message":"@typescript-bot in #34534: \u003c!--header {\"slug\":\"welcome\"} headerend--\u003e\r\n@chrisgedrim Thank you for submitting this PR!\n\n🔔 @stpettersens @yhaskell - please [review this PR](<a href="https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534/files">https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534/files</a>) in the next few days. Be sure to explicitly select **`Approve`** or **`Request Changes`** in the GitHub UI so I know what's going on.\n\nIf no reviewer appears after a week, a DefinitelyTyped maintainer will review the PR instead.\r\n\u003c!--footer bot @ 4/6/2019, 3:22:25 PM footerend--\u003e"}],"action":{"name":"View Pull Request","url":"<a href="https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421">https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421</a>"}}} [ { "@context": "<a href="http://schema.org">http://schema.org</a>", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "<a href="https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421">https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421</a>", "url": "<a href="https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421">https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34534#issuecomment-480542421</a>", "name": "View Pull Request" }, "description": "View this Pull Request on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "<a href="https://github.com">https://github.com</a>" } } ]

**Review by stpettersens — APPROVED** [review]
Looks good to me. 👍

**Review by yhaskell — APPROVED** [review]
Looks good to me too

File: types/express-handlebars/index.d.ts [hunk 1]
```diff
@@ -36,7 +40,7 @@ interface Exphbs {
     getTemplate(filePath: string, options?: PartialTemplateOptions): Promise<Function>;
     getTemplates(dirPath: string, options?: PartialTemplateOptions): Promise<Object>;
     render(filePath: string, context: Object, options?: RenderOptions): Promise<string>;
-    renderView(viewPath: string, optionsOrCallback: any, callback?: () => string): void;
+    renderView(viewPath: string, optionsOrCallback: ExphbsCallback | any, callback?: ExphbsCallback): void;
```
Comment (DanielRosenwasser) [inline types/express-handlebars/index.d.ts]: There's no point to this change if this is getting union-ed with `any`. Maybe you should create an overload instead.

```ts
renderView(viewPath: string, callback: ExphbsCallback): void;
renderView(viewPath: string, options: any, callback: ExphbsCallback): void;
```

Comment (chrisgedrim) [reply types/express-handlebars/index.d.ts]: And that's why you shouldn't code while tired 😂

Comment (DanielRosenwasser) [reply types/express-handlebars/index.d.ts]: Do you happen to be a medical professional? I'm constantly tired and I'm hoping to get  a doctor's note out of work 😂

**Pull request merged**